src/index.ts

```diff
@@ -320,8 +321,8 @@ export default runExtension(async ({ extensionAPI }) => {
         if (targetName) {
           throw new Error(`Could not find page with name ${targetName}`);
         } else {
-          throw new Error(
-            "Either the `targetName` or `targetUid` input is required"
+          return new Promise((resolve) =>
+            processBlockUid(srcUid).then(resolve)
```

Comment:
We want:
- `resetContext({ targetUid: null, variables, triggerUid });`
- `const childNodes = getFullTreeByParentUid(srcUid).children`
- `processChildren({ nodes: childNodes })`